posts/bayesian-model-comparison/index.qmd

```diff
@@ -0,0 +1,470 @@
+---
+title: Bayesian comparison of cross-validated algorithms
+subtitle: 'Model selection via a Bayesian correlated t-test.'
+date: 2022/11/11
+image: thumb.png
+categories:
+  - ML
+  - Theory
+  - Model evaluation
+  - Bayes
+  - Bayesian estimation
```

Comment:
I would just keep Bayes, or are we thinking the more the merrier?

Reply:
Definitely not more the merrier. I wasn't sure and these admittedly are mostly randomise placeholders. "Bayes" however makes me itch a little as we're not really talking about the man himself. "Bayesian"?